react-components/src/components/RevealToolbar/SettingsContainer/HighFidelityContainer.tsx

```diff
@@ -7,7 +7,7 @@ import { Menu } from '@cognite/cogs.js';
 import { useReveal } from '../../RevealContainer/RevealContext';
 import { type QualitySettings, type QualityProps } from './types';
 import { type Cognite3DViewer } from '@cognite/reveal';
-import { useTranslation } from '../../../common/i18n';
+import { useI18n } from '../../i18n/I18n';
```

Comment:
Can we just keep the name `useTranslation`? It is a little more obvious what it means, instead of `useI18n`.